scripts/plugins/__init__.py

```diff
@@ -1,3 +1,4 @@
 import logging
 
+
```

Comment:
What's going on here?

Reply:
We need 2 lines after import to conform with [PEP8](https://pep8.org).
See https://blog.finxter.com/when-to-add-two-blank-lines-in-python/